app/js/controllers/angel.js

```diff
@@ -0,0 +1,253 @@
+        }
+        
+        startupslocation.push(ele.locations[0].display_name);
+      }
+      
+      })
+      Geocode();
+
+        }, function(response) {
+    
+   
+    // called asynchronously if an error occurs
+    // or server returns response with an error status.
+  });
+
+
+    }
+   
+    
+    function Geocode()
+            { console.log("called");
+                
+                var locarray = {
+
+                    "places" : startupslocation
+                }
+               
+                
+            $http.jsonp('http://localhost:9000/api/geocode.json?callback=JSON_CALLBACK&minified=true', {params : { data : locarray } })
+            .success(function(data, status, headers, config) {
+              
+              var startupsMarker = {
+                "type": "FeatureCollection",
+                "features": []
+            };
+                //console.log( followers.propic[i]);
+                $scope.startups_status=startupslocation.length;
+                for(var i=0;i<startupslocation.length;i++)
+                {
+                    
+                   
+                    var locationkey=startupslocation[i];
+                    
+                    if(data.locations[locationkey].mark)
+                    {
+                      
+                    var textpopup=MapPopUpTemplateService.startupInfoPopUp($scope.startups[i]);
+                    //console.log("I am viewing "+followers[i].name);
+                    //followers.preview[i]=textpopup;
+                    var pointObject = {
+                        "geometry": {
+                            "type": "Point",
+                            "coordinates": [
+                                data.locations[locationkey].mark[0],
+                                data.locations[locationkey].mark[1]
+                            ]
+                        },
+                        "type": "Feature",
+                        "properties": {
+                            "popupContent" : "<div class='foobar'>"+textpopup+"</div>" ,
+                            "propic" : $scope.startups[i].logo_url
+
+                        },
+                        "id": "123"
+
+                    };
+                    startupsMarker.features.push(pointObject);
+
+                    }
+                }
+                   add_marker(startupsMarker);
+                    
+                
+                }).error(function(data, status, headers, config) {
+                    
+
+                    console.log("There is error.Loklak Server did not respond with geodata.We will try again."+data+"fff"+status);
+                    $scope.followers_status="Load Failed.Twitter did not respond."
+                    
+                        // called asynchronously if an error occurs
+                        // or server returns response with an error status.
+            });
+               
+        }
+
+  }
+
+
+  function add_marker(result) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'add_marker' is not in camel case.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2550181458/issues/source?bid=3288768&fileBranchId=3288808#l215)